### DISCUSSION TITLE
Disable crossover in NSGA3

### DISCUSSION THREAD

**Discussion by faranak1991** [anyoptimization/pymoo#601] [Q&A]
Hello,

I want to disable the crossover in the NSGA3 algorithm and only use my custom Mutation operator, after choosing  crossover=None in the algorithm:

algorithm = NSGA2(
    pop_size=population,
    sampling=random_population,
    crossover=None,
    mutation=CustomMutation(),
    n_offsprings=population,
    verbose=True,
    eliminate_duplicates=True )    

It shows this error :
 in _do
    n_matings = math.ceil(n_offsprings / self.crossover.n_offsprings)
AttributeError: 'NoneType' object has no attribute 'n_offsprings'

Any help to address this issue will be appreciated ?

**Comment by blankjul** [comment]
You can use `NoCrossover()` to just copy individuals from the population without any change.
